hydra-cluster/test/Test/CardanoNodeSpec.hs

```diff
@@ -21,7 +21,7 @@ supportedNetworks :: [KnownNetwork]
 supportedNetworks = [Sanchonet]
```

Comment:
This version should be supported on all networks.

Must: test against all networks.

Reply:
Done